typescript/docs/tutorials.mdx

```diff
@@ -0,0 +1,148 @@
+# BeeAI Framework Tutorials
```

Comment:
There is already a change in this PR which moves typescript/docs/tutorials.md to a mdx in the docs-mintlify tree.

Was the intent here to retain a 'local' version for people browsing the source? If so I presume it would be a .md not .mdx?
(Ok to fix in later pr if needed)